src/spawn.mli

```diff
@@ -0,0 +1,17 @@
+(** Subset of https://github.com/janestreet/spawn/blob/master/src/spawn.mli *)
```

Comment:
If that's easier to maintain, we can also vendor the actual lib in `vendor/`. (that can be done in a separate PR)

Reply:
Yh, I didn't do it yet as it has some C code so we need to be a bit careful about it